src/NuGet.Clients/PackageManagement.UI/DetailControlModel.cs

```diff
@@ -12,6 +12,7 @@
 using NuGet.ProjectManagement.Projects;
 using NuGet.Protocol.VisualStudio;
 using NuGet.Versioning;
+using System.Windows.Media.Imaging;
```

Comment:
adding some coding style comments - We can probably handle them ourselves.

System should go on top (fix default sorting order in vs for this)